### PR TITLE
tikv-ctl: implement bad regions for raftstore-v2

### PR DESCRIPTION
**Pull request by SpadeA-Tang**
### What is changed and how it works?

Issue Number: Ref #14654

What's Changed:

```commit-message
implement bad regions for raftstore-v2
```

### Related changes

- PR to update `pingcap/docs`/`pingcap/docs-cn`:
- Need to cherry-pick to the release branch

### Check List 

Tests 

- Unit test

### Release note 

```release-note
None
```

### CODE REVIEW

File: src/server/debug2.rs [hunk 1]
```diff
@@ -739,6 +814,16 @@ mod tests {
         DebuggerImplV2::new(reg, raft_engine, ConfigController::default())
     }
 
+    impl<ER: RaftEngine> DebuggerImplV2<ER> {
```
Comment (bufferflies) [inline src/server/debug2.rs:817]: only used for test?

Comment (SpadeA-Tang) [reply src/server/debug2.rs:817]: yes

File: components/raftstore-v2/src/operation/command/admin/flashback.rs [hunk 2]
```diff
@@ -85,6 +85,7 @@ impl<EK: KvEngine, R: ApplyResReporter> Apply<EK, R> {
 
 impl<EK: KvEngine, ER: RaftEngine> Peer<EK, ER> {
     // Match v1 on_set_flashback_state.
+    #[allow(unused_mut)]
```
Comment (overvenus) [inline components/raftstore-v2/src/operation/command/admin/flashback.rs:88]:
```suggestion
```

There is an `allow` at line 92.

**Review by overvenus — COMMENTED** [review]
Rest LGTM

File: components/raftstore-v2/src/operation/command/admin/flashback.rs (hunk 2 above)
Comment (SpadeA-Tang) [reply components/raftstore-v2/src/operation/command/admin/flashback.rs]: Good catch.

**Review by overvenus — APPROVED** [review]

**Review by bufferflies — APPROVED** [review]

**Pull request merged**